lib/lambda-worker/lambda-worker-props.ts

```diff
@@ -0,0 +1,54 @@
+import * as cdk from "@aws-cdk/core";
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as iam from "@aws-cdk/aws-iam";
+import * as sns from "@aws-cdk/aws-sns";
+
+export interface LambdaWorkerProps {
+  // The name of the LambdaWorker
+  name: string;
+
+  // Lambda Properties
+  // Documented here https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda-nodejs.NodejsFunctionProps.html
+  lambdaProps: {
+    description?: string;
+    handler: string;
+    entry: string;
+    environment?: { [key: string]: string };
+    memorySize: number; // LambdaWorker will set a minimum memory size of 1024
+    reservedConcurrentExecutions?: number;
+    retryAttempts?: number;
+    role?: iam.IRole;
+    securityGroup?: ec2.ISecurityGroup;
+    timeout: cdk.Duration;
+    vpc?: ec2.IVpc;
```

Comment:
```suggestion
    vpc: ec2.IVpc;
```

Reply:
It's not for us, we (aim to) deploy everything in a VPC.

Reply:
New AWS accounts require everything to be in a VPC, and EC2 classic will be retired next August. https://aws.amazon.com/blogs/aws/ec2-classic-is-retiring-heres-how-to-prepare/